### ISSUE TITLE
Is the readme.md up to date? 

### ISSUE THREAD

**Issue by Kerrialn** [Kerrialn/oauth2-example-project#3]
@johnvanham did you update the readme to reflect the changes of how the package implemented?

**Comment by johnvanham** [comment]
@Kerrialn not yet - I've put a TODO at the top and I can sort that out when I get a moment if you like..

I thought there could be one section on how to set up and try out the example application from this repo, and then a section on how to set up the oauth2 bundle for the authorization code grant in any Symfony project. What do you think?

**Comment by Kerrialn** [comment]
Yeah sounds good, I'll create a code clean up branch and run some code checks.

**Comment by Kerrialn** [comment]
Ok, cleaned up, you can check it here: #4

**Comment by Kerrialn** [comment]
@johnvanham any updates on this? Next thing I want to add is how to use Oauth to register / login with social, like fb, google, twiter, etc.

**Comment by johnvanham** [comment]
Hi @Kerrialn sorry, been incredibly busy both with work and outside of work.

I've now got this working in production with a some custom authenticators in the mix, for the user authentication/login as we needed it to work with a user database from a Wordpress site and also work alongside a basic API key integration. We've yet to have 3rd parties use the oAuth part though but that should be imminent and should prove if it's all been set up right!

For the register/login with social, would that be for the user authentication side? So you it would be both an oAuth2 server plus oAuth2 client?

I guess that would require adding another bundle? I think I've seen a few Symfony 5/6 oauth2 client bundles with Google etc. already built-in so I expect you'll find that easier than using the oauth2-server-bundle :-) Will you do that as a separate repository or a separate branch?
